test/e2e/nightwatch.config.js

```diff
@@ -7,7 +10,7 @@ module.exports = {
 
   'selenium': {
     'start_process': true,
-    'server_path': 'node_modules/selenium-server/lib/runner/selenium-server-standalone-2.53.1.jar',
+    'server_path': seleniumServer.path,
```

Comment:
To be consistent we should do `server_path': require('selenium-server').path` instead